src/pkg/selectors/groups.go

```diff
@@ -334,9 +404,16 @@ const (
 
 	// details.itemInfo comparables
 
-	// channel drive selection
+	// channel and drive selection
 	GroupsInfoSiteLibraryDrive groupsCategory = "GroupsInfoSiteLibraryDrive"
 	GroupsInfoChannel          groupsCategory = "GroupsInfoChannel"
+
+	// data contained within details.ItemInfo
+	GroupsInfoChannelMessageCreatedAfter    groupsCategory = "GroupsInfoChannelMessageCreatedAfter"
+	GroupsInfoChannelMessageCreatedBefore   groupsCategory = "GroupsInfoChannelMessageCreatedBefore"
+	GroupsInfoChannelMessageCreator         groupsCategory = "GroupsInfoChannelMessageCreator"
+	GroupsInfoChannelMessageLastReplyAfter  groupsCategory = "GroupsInfoChannelMessageLastReplyAfter"
+	GroupsInfoChannelMessageLastReplyBefore groupsCategory = "GroupsInfoChannelMessageLastReplyBefore"
```

Comment:
In future we may be able to add more selectors like - filter on basis of 
- messages with events
- members in channel

Reply:
True, but no need to get ahead of ourselves.  We haven't identified a use case for these properties yet.  When we figure out the UX needed, we can revisit their addition.  The properties I've added are table stakes, as identified by Georgi and I in conversation.